data/heliarch intro.txt

```diff
@@ -0,0 +1,1568 @@
+			`	"To combat these thoughts of disunion, we swore. Correct such behavior, we must."`
+			`	"However dedicated, our agents and ships are, predictable and slow, often times we are. A different approach, might be due. An outsider's help, their way of thinking, we'd like to ask for."`
+			choice
+				`	"What exactly would I have to do?"`
+				`	"In that case I'm not interested, sorry. I thought this was some simple job, I don't want to get involved in such matters."`
+					decline
+			`	"Six worlds, we have marked. Six worlds, we'd like you to 'visit'. Those three tourists, you will be transporting," an Arach says as they point to Iplora, Kirrie and Tugroob.`
+			choice
+				`	"Sounds easy enough, where should I start?"`
+				`	"Whether they go undercover or not, this seems a bit risky for me. Sorry, you'll have to find someone else."`
+					decline
+			`	"Your choice, that is," Iekie responds. "Known to the rebels, many of our methods are. Easily predicted, our order of worlds would be."`
+			`	Some of the others explain that you will have to visit the following worlds they have deemed suspicious: Shifting Sand, in Saryd space, Into White, Fourth Shadow and Remote Blue, in Kimek space, Mebla's Portion and Stronghold of Flugbu, in Arach space.`
+			`	They wish the four of you luck, and you're escorted back to the spaceport, after which you show the 'tourists' to their bunks in your ship.`
+				accept
+	on complete
+		"assisted heliarch" ++
+		log "Helped the Heliarchs investigate several worlds around Coalition space, bringing Heliarch agents to each world so that they could investigate signs of Resistance activity."
+		payment 900000
+		conversation
+				branch evidence0
+					"resistance evidence" == 0
+				branch evidence1
+					"resistance evidence" == 1
+				branch evidence2
+					"resistance evidence" == 2
+				branch evidence3
+					"resistance evidence" == 3
+			label evidence0
+			`The Heliarch elder, Iekie, greets you and the agents when you return, and gives you your payment of 900000 credits.`
+			`	"Unearthed some useful information, we have," Iplora tells her. "Essential to the mission's success, Captain <last>'s cooperativeness was."`
+			`	"No doubt of that, I have. To the report, looking forward I am."`
+			`	They thank you once again for helping, and then walk to the Heliarch-only parts of the ringworld.`
+				accept
+			label evidence1
+			`The Heliarch elder, Iekie, greets you and the agents when you return, and gives you your payment of 900000 credits.`
+			`	"Unearthed some useful information, we have," Iplora tells her. "Essential to the mission's success, Captain <last>'s cooperativeness was."`
+			`	"No doubt of that, I have. To the report, looking forward I am."`
+			`	They then get closer to her, and Iplora whispers something, handing her a separate report card, written differently than the Coalition languages that you've seen. It must be some type of code.`
```

Comment:
`	The agents then get closer to her before Iplora whispers something, handing her a separate report card that's written differently than the Coalition languages you've seen. It must be some type of code.`